tests/test_stl.cpp

```diff
@@ -61,6 +61,50 @@ TEST_SUBMODULE(stl, m) {
         return set.count("key1") && set.count("key2") && set.count("key3");
     });
 
+    #if defined(_MSC_VER) && _MSC_VER < 1910
+    // We get some really long type names here which causes MSVC 2015 to emit warnings
+    #  pragma warning(disable: 4503) // warning C4503: decorated name length exceeded, name was truncated
```

Comment:
I'd suggest adding this globally to `pybind11_tests.h` since it's already required in two files and likely more in the future.